src/Faker/Provider/kk_KZ/Person.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Faker\Provider\kk_KZ;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{lastName}}ұлы {{firstNameMale}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{lastName}}қызы {{firstNameFemale}}',
+    );
+
+    protected static $firstNameMale = array(
```

Comment:
Could you add a link in the phpDoc to the source of this data? Same for other Person formatters